selinux/cockpit.te

```diff
@@ -19,6 +19,7 @@ userdom_user_tmpfs_file(cockpit_tmpfs_t)
 type cockpit_var_run_t;
 files_pid_file(cockpit_var_run_t)
 systemd_mount_dir(cockpit_var_run_t)
+systemd_private_tmp(cockpit_var_run_t)
```

Comment:
FYI, this will not take effect on RHEL/CentOS 8 right now. See PR #16575, if/when this lands, this will Just Work™, otherwise we'll need to backport this policy to the central RHEL 8 one, and apply a local workaround for the time being.

Reply:
#16575 landed, so resolving. This should work on next rebase/test run.